src/config.json

```diff
@@ -799,6 +799,356 @@
     "satoshilabs.design"
   ],
   "blacklist": [
+    "git.access.sstqkeqamble.com",
+    "gitlab.access.sstqkeqamble.com",
+    "direct.traderjjoe.xyz",
+    "access.sstqkeqamble.com",
+    "dragonrichclub.co",
+    "joinvalhalla.nftmintlive.com",
+    "dudulab-nft.com",
+    "osf.glnfts.xyz",
+    "tribe.gwnfts.xyz",
+    "castaways.glnfts.xyz",
+    "doodles.glnfts.xyz",
+    "devbear.glnfts.xyz",
+    "neotokyo.glnfts.xyz",
+    "keungz.glnfts.xyz",
+    "azuki.glnfts.xyz",
+    "moonbirds.glnfts.xyz",
+    "pegz.gwnfts.xyz",
+    "pgodjira.glnfts.xyz",
+    "digidaigaku.glnfts.xyz",
+    "treeverse.gwnfts.xyz",
+    "rtfkt.gwnfts.xyz",
+    "wheremyvansgo.gwnfts.xyz",
+    "boredapeyachtclub.gwnfts.xyz",
+    "ghxsts.gwnfts.xyz",
+    "cyberkongz.gwnfts.xyz",
+    "proof.gwnfts.xyz",
+    "0xmons.gwnfts.xyz",
+    "www.metaninjasnft.org",
+    "testnet.gamefiindonesia.com",
+    "www.0xazuki.io",
+    "themintpage.com",
+    "www.sushiblocks.com",
+    "crpc-mint.com",
+    "chaintoadz.com",
+    "www.metaclubmint.live",
+    "mintloz.michaelbisbell.com",
+    "mintloz.com",
+    "git.git.git.gitlab.gitlab.sitemaps.goblintown-wtf.com",
+    "gitlab.gitlab.git.gitlab.gitlab.git.sitemaps.goblintown-wtf.com",
+    "gitlab.git.git.gitlab.gitlab.git.sitemaps.goblintown-wtf.com",
+    "sakebaro.com",
+    "dmitriychernyaknft.art",
+    "www.seabums.xyz",
+    "dinoclub.org",
+    "nft-mint-page.pages.dev",
+    "xn--starnet-m3b.com",
+    "naughtyteddies-club.worldcuriositie.com",
+    "mintroyalsdeldesierto.xyz",
+    "garbfriends.com",
+    "10ktf-otherside.com.premint-register.xyz",
+    "otherside-vials.xyz.premint-register.xyz",
+    "www.driftqueensmint.com",
+    "bonusmint.xyz",
+    "cowsvsaliens.nftminters.io",
+    "meta-legends.tech",
+    "nftmonamour.fr",
+    "git.gitlab.git.git.gitlab.sitemaps.goblintown-wtf.com",
+    "sandbox.traderjjoe.xyz",
+    "esion-mint.com",
+    "skulltoons.mintsystems.io",
+    "www.flipprtoolmint.online",
+    "www.flipprtoolmint.newgems.xyz",
+    "babyghosts-premint.xyz",
+    "www.seabumsnft.nftsmint.site",
+    "metakozo-dao.free-mints.icu",
+    "gitlab.git.git.git.git.gitlab.sitemaps.goblintown-wtf.com",
+    "git.git.gitlab.gitlab.gitlab.sitemaps.goblintown-wtf.com",
```

Comment:
Thank you for your contribution!

There seem to be a lot of redundancies here where a higher-level domain can be considered malicious. (Shared hosting and such aside, of course). `sitemaps.goblintown-wtf.com` seems recurring enough, for example.

Reply:
Thank you for your response, I would remove the redundant entries and commit them again.